media/pages/sk/faq/page.html

```diff
@@ -5,108 +5,181 @@ <h1>Často kladené otázky</h1>
 
 <h2 id="o-portali">O portáli</h2>
 
-<h3 id="co-je-chcemvediet-sk">Čo je chcemvediet.sk?</h3>
-<p>
-Chcemvediet.sk je portál, prostredníctvom ktorého môžete poslať žiadosť o informácie ktorejkoľvek
-inštitúcii, ktorá má povinnosť sprístupňovať informácie v zmysle zákona o slobodnom prístupe
-k informáciám č. 211/2000 Z.z. z jedného miesta, a ďalej spravovať svoje žiadosti o informácie.
-Zároveň môžete v databáze vyhľadávať medzi informáciami, ktoré už boli sprístupnené iným žiadateľom
-a sledovať štatistiky o vybavovaní žiadostí o informácie.
-</p>
+<div>
+  <h3 id="co-je-chcemvediet-sk">
+    <a data-toggle="collapse" href="#co-je-chcemvediet-sk-text">
+      <i class="chv-icon chv-icon-lg icon-plus"></i>
+    </a>
+    Čo je chcemvediet.sk?
+  </h3>
+  <p id="co-je-chcemvediet-sk-text" class="collapse">
+  Chcemvediet.sk je portál, prostredníctvom ktorého môžete poslať žiadosť o informácie ktorejkoľvek
+  inštitúcii, ktorá má povinnosť sprístupňovať informácie v zmysle zákona o slobodnom prístupe
+  k informáciám č. 211/2000 Z.z. z jedného miesta, a ďalej spravovať svoje žiadosti o informácie.
+  Zároveň môžete v databáze vyhľadávať medzi informáciami, ktoré už boli sprístupnené iným
+  žiadateľom a sledovať štatistiky o vybavovaní žiadostí o informácie.
+  </p>
+</div>
 
-<h3 id="preco-chcemvediet-sk">Prečo chcemvediet.sk?</h3>
-<p>
-Lebo si myslíme, že právo na informácie má mať naozaj každý. A každý ho má naozaj len vtedy, keď
-rozumie aj bez pomoci odborníka, aké informácie a od koho môže žiadať, a ako má postupovať, keď mu
-inštitúcia odmietne sprístupniť požadované informácie.
-</p>
+<div>
+  <h3 id="preco-chcemvediet-sk">
+    <a data-toggle="collapse" href="#preco-chcemvediet-sk-text">
```

Comment:
Z nejakeho dovodu animacia nie je plynula. Ale na konci animacie text trochu podskoci. Zislo by sa zistit preco a spravit ju plynulu.

Reply:
:ok: @martinmacko47 element ktory sa zobrazuje ako collapse nemoze mat padding. Da sa to fixnut zabalenim collapse obsahu do samostatneho `<div>`-ka.